kolibri/core/auth/management/commands/bulkexportusers.py

```diff
@@ -203,6 +207,17 @@ def csv_file_generator(facility, filepath, overwrite=True):
                 usernames.add(item["username"])
             yield item
 
+        f.seek(0)
+        file = file_storage.save(filename, f)
+
+        try:
+            # If the file is local, we can get the path
+            logger.info("File saved - Path: {}".format(file_storage.path(file)))
+        except NotImplementedError:
+            # But if path is not implemented, we assume we can get the URL
+            logger.info("File saved - Path: {}".format(file_storage.url(file)))
+            logger.info("File saved - Size: {}".format(file_storage.size(file)))
```

Comment:
@jredrejo I've tried several things around here, but no matter what I do I the file always shows size 0... I've confirmed that there are users (usernames is full of data, for example) -- so I'm not sure why the `writer` isn't updating the file object here...

Reply:
The problem was not in reading the file info, but writing it. It had 0 bytes. I have made a commit in this PR to ensure data is flushed into the BytesIO and now it's working:
```
INFO     2024-12-26 20:45:45,313 Invoking command bulkexportusers
INFO     2024-12-26 20:45:45,373 Creating users csv file /home/jose/.kolibri/log_export/Kolibri en casa de admin_3da4_users.csv
  [#######################################################################################################################################################################################-------------------------------------------------------------]   75%INFO     2024-12-26 20:45:46,767 File saved - Path: https://storage.googleapis.com/kdp-csv-reporting-develop/Kolibri_en_casa_de_admin_3da4_users.csv
INFO     2024-12-26 20:45:47,073 File saved - Size: 482
INFO     2024-12-26 20:45:47,073 Created csv file /home/jose/.kolibri/log_export/Kolibri en casa de admin_3da4_users.csv with 4 lines
```
(I've also rebased develop in the branch because there was already a conflict and more might appear soon)